### PR TITLE
Missing destruction of Omega

### PR DESCRIPTION
**Pull request by ndevelder**
Hi All! 

A couple fixes to RANS model here. Seems like there was a typo in the omega src term? 

Also, if I print out "wd_arr" during a kOmegaSST sim, I get only zeros, and I can't seem to find anywhere that we're computing wall distance. This is important to the blending behavior of k-Omega SST. Maybe we should try and fix that on this PR as well?

### CODE REVIEW

**Review by psakievich — APPROVED** [review]

**Comment by ndevelder** [comment]
Just tested this "fix" on the channel flow regression test and it blows up quite quickly. Let's hold off on merging until we get a comment from Ganesh and do some more troubleshooting?

**Comment by marchdf** [comment]
@ndevelder I noticed the lack of wall distance calculation in amr-wind as well. This is most likely the reason why this source term was turned off. It would be great to have a wall distance calculator but I think that's a bunch of work... We can see what we can port from nalu-wind to make this happen.

**Comment by psakievich** [comment]
For just amr-wind the wall distance is pretty trivial right? You don't really need an elliptic solve. I think we would need to do a lot of work to get it working with the hybrid solver for lots of reasons.  But for pure amr-wind I think it is just boundary condition logic and some integer math.  For IB it is already available through the level set.

**Comment by marchdf** [comment]
Yeah that totally makes sense for pure amr-wind. I was thinking about the hybrid solver stack and what I want to do there. But that's not needed for this.

**Review by psakievich — CHANGES_REQUESTED** [review]
Putting a blocker on this.

**Comment by gantech** [comment]
I think we should merge this right away even if the wall distance issue is not fixed. This is important for hybrid RANS simulations of wind turbines right now

**Comment by psakievich** [comment]
@ndevelder do you want to comment on this?  This initial fix was unstable correct?

**Comment by ndevelder** [comment]
I'd vote for committing this with the assumption that the physics type or a new version of the turbulence model will handle the wall distance correctly (even though we know this isn't always true?) It's clearly a bug, but I'd rather have it crash or go unstable than never show up in the simulation?

**Review by psakievich — APPROVED** [review]

**Pull request merged**
